setup.py

```diff
@@ -28,17 +28,19 @@
     print(error, file=sys.stderr)
     sys.exit(1)
 
-from distutils.core import setup
-import distutils.dir_util
+from setuptools import setup, Extension, Command
+
+import warnings
+
 import os
 import subprocess
 import fileinput
 
+import re
+
+setup_path = os.path.dirname(__file__)
+
 import hyperspy.Release as Release
-# clean the build directory so we aren't mixing Windows and Linux
```

Comment:
how that is even possible? There is `setup.py clean` command to do cleaning, and clean up is also now implemented with git post-checkout hooks